app/src/main/java/io/homeassistant/companion/android/sensors/NetworkSensorManager.kt

```diff
@@ -177,11 +177,11 @@ class NetworkSensorManager : SensorManager {
         val settingName = "replace_$bssid"
```

Comment:
This is also a sensor setting for the sensor if your intention is to translate all of settings

Reply:
Yes you are right. Should be also translated.